### ISSUE TITLE
Show Activity Output in Execution Log in Workflow Instance Viewer

### ISSUE THREAD

**Issue by sfmskywalker** [elsa-workflows/elsa-core#1136]
With the update introducing Output Storage provider, the information about output of an activity has disappeared from the Journal. We need to add this back.

For output stored in the workflow it's easy, but care needs to be taken to prevent loading of large blobs from the blob storage provider for example.

**Comment by sfmskywalker** [comment]
To avoid too much data duplication in the workflow execution log table, I am considering dropping output logging altogether and instead just log the actual outcome of an activity. The Activity State tab will show all activity input & output, which is stored only once and (almost) as useful as being able to see the input & output on an individual workflow execution log record.

If someone does need the exact information used at the time of execution (for example when activities execute multiple times and you want to be able to see what input got in and output got out), the application can always handle the `ActivityExecuted` event and add log entries using `IWorkflowJournal`. This gives precise control over what to log when.

**Comment by sfmskywalker** [comment]
We still do want to be able to access output that is stored elsewhere (such as blob storage) using the hyperlink solution, but I will create a separate issue for that.
